src/controllers/controller.bubble.js

```diff
@@ -168,6 +149,7 @@ BubbleController.defaults = {
 	plugins: {
 		tooltip: {
 			callbacks: {
+				_scriptable: false,
```

Comment:
Only for that kind of situations: https://github.com/chartjs/Chart.js/blob/650956b2e1ec8a9591135476cc61f64ddaad7f12/test/specs/plugin.legend.tests.js#L711

That is the only test that actually detects it, but it would apply to tooltip callbacks too if the plugin defaults are removed.

Reply:
It might be better to relocate these "descriptors" so they are not removed when defaults are modified and that would remove the need to replicate.

Reply:
Yeah, i can see if causing problems. Not sure where the best place to move it would be.

Reply:
I moved the descriptors to a private symbol in defaults and added that as a final fallback scope for resolution. Thoughts?

Reply:
Good idea, makes sense to me and will prevent errors